esp32/modules/tasks/powermanagement.py

```diff
@@ -16,9 +16,9 @@ def pm_task():
     global requestedStandbyTime
     
     idleTime = virtualtimers.idle_time()
-    print("[Power management] Next task wants to run in "+str(idleTime)+" ms.")
+    #print("[Power management] Next task wants to run in "+str(idleTime)+" ms.")
         
-    if idleTime>30000:
+    if idleTime > 30000 and ( badge.usb_volt_sense() < 4500 or badge.nvs_get_u8('badge', 'usb_stay_awake', 0) == 0 ):
```

Comment:
Is this really the best way we have of determining whether the device is plugged into USB?